src/3p-frame.js

```diff
@@ -60,18 +60,36 @@ function getFrameAttributes(parentWindow, element, opt_type, opt_context) {
   return attributes;
 }
 
+/** @record */
+class IframeOptions {
```

Comment:
we usually just define the type in place ` @param {{disallowCustom:..}} options` or use typedef and name them `/** @typedef {{disallowCustom:..}} */ let IframeOptionsDef;`

Reply:
Changed to just inline it. The one limitation is that you can't comment the parameters :(. Maybe better parameter names would help?